run_pipeline.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-./packages/cli/bin/index.js run ./test/pipelines/$1.json
+./packages/cli/dist/bin/index.js run ./test/pipelines/$1.json
```

Comment:
Does the chmod need?

Reply:
I run chmod in cli/package.json:scripts, so not need here.